frontend/src/pages/Konva.js

```diff
@@ -0,0 +1,158 @@
+import React, {useState} from 'react';
+import {Stage, Layer, Rect, Circle} from 'react-konva';
+
+export default function Konva() {
+    const [rects, setRects] = useState([]);
+    const [circles, setCircles] = useState([])
+    const [rectId, setRectId] = useState(1)
+    const [circleId, setCircleId] = useState(1)
```

Comment:
This states and the following functions just scream to be refactored into a custom hook :) https://reactjs.org/docs/hooks-custom.html (-> extra ticket, fine for now)